src/logics/SubstLoopBreaker.cc

```diff
@@ -81,7 +81,7 @@ TargetVarList::TargetVarList(vec<PTRef>&& _children)
 
 TVLRef TargetVarListAllocator::alloc(vec<PTRef>&& _children)
 {
-    sort(_children);
```

Comment:
Why do these need to be sorted?
Does it matter how they are sorted (ascending/descending)?

Reply:
I doubt the order matters.  I vaguely remember it being some sort of a requirement for termination, but I'm not sure.